Error: prefix already exists: /home/ubuntu/miniconda/envs/_pipbuild_
Exception:
Traceback (most recent call last):
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/basecommand.py", line 223, in main
    status = self.run(options, args)
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/commands/install.py", line 268, in run
    wheel_cache
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/basecommand.py", line 268, in populate_requirement_set
    wheel_cache=wheel_cache
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/req/req_install.py", line 207, in from_line
    wheel_cache=wheel_cache)
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/req/req_install.py", line 66, in __init__
    req = pkg_resources.Requirement.parse(req)
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2960, in parse
    reqs = list(parse_requirements(s))
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2904, in parse_requirements
    "version spec")
  File "/home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2869, in scan_list
    raise ValueError(msg, line, "at", line[p:])
ValueError: ('Expected version spec in', 'zope.interface==>=4.1.0', 'at', '==>=4.1.0')

Package search: {'_pypi_ordering': False, 'version': '2.1.2', 'name': 'plone.app.linkintegrity', 'summary': 'Manage link integrity in Plone.'}
Creating standard recipe for plone.app.linkintegrity-2.1.2
Converting recipe in /home/ubuntu/pypi-conda-builds/plone.app.linkintegrity
package = plone.app.linkintegrity
   dependencies = [u'plone.app.referenceablebehavior']
Creating standard recipe for plone.app.referenceablebehavior
Converting recipe in /home/ubuntu/pypi-conda-builds/plone.app.referenceablebehavior
package = plone.app.referenceablebehavior
   dependencies = [u'plone.behavior', u'plone.dexterity >=1.1', u'plone.indexer', u'plone.uuid', u'products.archetypes']
Creating standard recipe for plone.behavior
Converting recipe in /home/ubuntu/pypi-conda-builds/plone.behavior
package = plone.behavior
   dependencies = [u'zope.component', u'zope.interface', u'zope.schema', u'zope.annotation', u'zope.configuration']
Creating standard recipe for zope.component
Converting recipe in /home/ubuntu/pypi-conda-builds/zope.component
package = zope.component
   dependencies = [u'zope.interface >=4.1.0', u'zope.event']
Creating standard recipe for zope.interface->=4.1.0
Error: Version >=4.1.0 of zope.interface is not available on PyPI.

conda create -n _pipbuild_ --yes python pip
/home/ubuntu/miniconda/envs/_pipbuild_/bin/pip install zope.interface==>=4.1.0
Error: Could not pip install zope.interface==>=4.1.0
